pymc/data.py

```diff
@@ -501,7 +500,7 @@ def determine_coords(model, value, dims: Optional[Sequence[str]] = None) -> Dict
         for size, dim in zip(value.shape, dims):
             coord = model.coords.get(dim, None)
             if coord is None:
-                coords[dim] = pd.RangeIndex(size, name=dim)
+                coords[dim] = range(size)
```

Comment:
not sure, but maybe it needs to be converted to a list?

Reply:
I think it's okay either way. In the end, the coords get turned into a tuple:

https://github.com/pymc-devs/pymc/blob/fb7e041fd08ad9c23e06c5ded2cac70a2ee0f9a5/pymc/model.py#L1105 

If it's the same, I slightly prefer `range` since it does not need to create the whole list here. (`pd.RangeIndex` stores a `range` internally)

A `tuple` will be created from the coords when it gets registered with a model.

Reply:
sounds good